### PR TITLE
Make trino-cli available in Trino test containers

### PR DESCRIPTION
**Pull request by wendigo**
This allows using `trino-cli` from environment started with env up:

`docker exec -ti ptl-presto-master /docker/trino-cli`

### CODE REVIEW

**Comment by findepi** [comment]
> `docker exec -ti ptl-presto-master /docker/trino-cli`

will this work for secured environments?

**Review by losipiuk — APPROVED** [review]

File: testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/cli/TestRun.java [hunk 1]
```diff
@@ -262,7 +257,6 @@ private Environment getEnvironment()
                 container
                         // the test jar is hundreds MB and file system bind is much more efficient
                         .withFileSystemBind(testJar.getPath(), "/docker/test.jar", READ_ONLY)
-                        .withFileSystemBind(cliJar.getPath(), "/docker/trino-cli", READ_ONLY)
```
Comment (kokosing) [inline testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/cli/TestRun.java:265]: why it was mounted here? Is it used in tests?

**Comment by kokosing** [comment]
> will this work for secured environments?

No. Best would be to have some kind of wrapper script to which each environment could contribute so we could connect to presto with kerberos etc.

**Comment by wendigo** [comment]
> > `docker exec -ti ptl-presto-master /docker/trino-cli`
> 
> will this work for secured environments?

Only if you provide proper arguments (I have an idea for that)

**Comment by findepi** [comment]
If it doesn't work for secured, i am not sure what is the benefit. Unsecured are easily accessible from outside.

And, we should reserve the place where we add the wrapper script necessary to expose CLI in the secured ones.

File: testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/cli/TestRun.java (hunk 1 above)
Comment (wendigo) [reply testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/cli/TestRun.java:265]: Yes. There are CLI tests that are using cli directly.

**Comment by wendigo** [comment]
@findepi this could be achieved easily with: https://github.com/trinodb/trino/pull/6566 (allow passing cli arguments with env variables (command line provided arguments still take precedence over defaults from env))

**Comment by wendigo** [comment]
Another take on providing CLI configuration for secure environments: https://github.com/trinodb/trino/pull/6569

**Comment by bitsondatadev** [comment]
:wave: @wendigo - this PR is inactive and doesn't seem to be under development, and it might already be implemented by the PR you referenced #6569. If you'd like to continue work on this at any point in the future, feel free to re-open.

**Pull request closed without merging**